json_theme/lib/src/codec/theme_decoder.dart

```diff
@@ -10219,10 +10220,13 @@ class ThemeDecoder {
   ///  * [decodeTabBarIndicatorSize]
   ///  * [decodeTextScaler]
   ///  * [decodeTextStyle]
-  static TabBarTheme? decodeTabBarTheme(dynamic value, {bool validate = true}) {
-    TabBarTheme? result;
+  static TabBarThemeData? decodeTabBarTheme(
```

Comment:
This would need to become `decodeTabBarThemeData` or else downstream code won't be able to properly detect it.